lib/flex_commerce_api/api_base.rb

```diff
@@ -112,18 +112,20 @@ def public_attributes
     end
 
     def meta_attribute(key)
-      if RELATED_META_RESOURCES.include?(attributes[:meta_attributes][key][:data_type])
-        self.send(key) rescue nil
-      else
-        attributes[:meta_attributes][key][:value] rescue nil
+      begin
+        return self.send(key) if RELATED_META_RESOURCES.include?(attributes[:meta_attributes][key][:data_type])
+        attributes[:meta_attributes][key][:value]
+      rescue
```

Comment:
Personally I hate rescues that just rescue anything - if absolutely ANY error occured, it returns nil ? This sort of thing has wasted a lot of time for me in the past as it hides real errors.  Can you not rescue from the error you are expecting ?